Explorer/Assets/Scripts/ECS/SceneLifeCycle/Systems/PartitionSceneEntitiesSystem.cs

```diff
@@ -43,7 +43,7 @@ public partial class PartitionSceneEntitiesSystem : BaseUnityLoopSystem
         private bool isRunningJob;
         private bool forceJobRun;
         private int currentPartitionIndex;
-        private readonly NativeArray<int> SqrDistanceBuckets;
+        private NativeArray<int> sqrDistanceBuckets;
```

Comment:
I wasn't sure about this, so I removed the `readonly` to avoid the warning

![image](https://github.com/decentraland/unity-explorer/assets/7646450/fdbede8b-2178-4905-91a9-efc50689d9e4)